### PR TITLE
Feature/pagination

### PR DESCRIPTION
**Pull request by marla-singer**
## Changes

1. Add pagination
3. Add comment for new code
4. Remove useless helper
5. Place the action buttons at right side
6. Change dependes of profile.name field to username field
7. Update README
8. Update package.js

### Testing

1. Create folder in your project with name **packages**
1. Move package to this folder
1. Do `meteor add brylie:accounts-admin-ui`
1. Replace this number to yours for fast testing pagination
https://github.com/marla-singer/meteor-accounts-admin-ui-bootstrap-3/blob/1ad4532c27ae58532615ac0bde11fa9c41ef8035/client/accounts_admin.js#L79

### CODE REVIEW

File: .versions [hunk 1]
```diff
@@ -7,7 +7,7 @@ blaze@2.1.2
 blaze-tools@1.0.3
 boilerplate-generator@1.0.3
 brylie:accounts-admin-ui@0.2.8
-brylie:first-admin@0.1.0
+brylie:first-user-admin@0.1.9
```
Comment (brylie) [inline .versions]: We should use `apinf:first-user-admin` here.

File: .versions [hunk 2]
```diff
@@ -21,6 +21,8 @@ http@1.1.0
 id-map@1.0.3
 jquery@1.11.3_2
 json@1.0.3
+kurounin:pagination@1.0.13
```
Comment (brylie) [inline .versions]: Cool package!

:smile_cat:

File: .versions (hunk 1 above)
Comment (marla-singer) [inline .versions]: It's only `apinf:first-admin` https://atmospherejs.com/brylie/accounts-admin-ui?q=apinf

File: package.js [hunk 3]
```diff
@@ -1,33 +1,43 @@
...
   api.versionsFrom("1.0.1");
-	api.use('standard-app-packages', ['client', 'server']);
-	api.use("alanning:roles@1.2.11", ['client', 'server']);
-	api.use("brylie:first-admin@0.1.0", ['server']);
+	api.use(["standard-app-packages",
+        "alanning:roles@1.2.11"
+        ], ["client", "server"]);
 
-	api.add_files('libs/user_query.js', ['client', 'server']);
+    api.use("brylie:first-user-admin@0.1.9", ["server"]);
```
Comment (brylie) [inline package.js]: Lets use `apinf:first-user-admin` instead.

**Comment by brylie** [comment]
This looks good. Lets make a couple of improvements:

- switch to `apinf:first-user-admin`
- add .eslintrc to project
  - lint files, e.g. indentation

File: .versions (hunk 1 above)
Comment (brylie) [inline .versions]: OK, we need to publish the second package.

What is your Atmosphere username?

Should these, instead, be NPM modules?

**Comment by brylie** [comment]
One other thing, this PR should actually be made against the Apinf package:
https://github.com/apinf/meteor-accounts-admin-ui-bootstrap-3

**Comment by marla-singer** [comment]
@brylie So close this PR and I will fork another PR

**Comment by brylie** [comment]
Jep. Your changes look good. We can worry about the linting and packages later.

**Comment by brylie** [comment]
In the long run, these packages should all be under the `apinf` namespace, and follow our coding guidelines.

**Comment by marla-singer** [comment]
Closed this PR like not relevant

**Pull request closed without merging**